extensions/ql-vscode/src/contextual/queryResolver.ts

```diff
@@ -178,6 +178,7 @@ export async function runContextualQuery(
     { queryPath: query, quickEvalPosition: undefined },
     false,
     getOnDiskWorkspaceFolders(),
+    undefined,
```

Comment:
Do we want the possibility of running with extension packs here? The internal CFG query could give better results with extension packs.

Reply:
How are the internal CFG query's results affected by extension packs?

Reply:
Since the graph is for internal use only (and only barely supported), we can probably avoid adding extension packs until someone asks for it.